### DISCUSSION TITLE
CUSTOM_MODELS不生效？

### DISCUSSION THREAD

**Discussion by 2u0ta0** [ChatGPTNextWeb/NextChat#4648] [General]
在配置页面，非常想只把我们可以提供的模型（后端接了oneapi）暴露给最终用户，否则即便他们选择了某个模型不能用也很影响感受。
在官方文档中看到了CUSTOM_MODELS这个参数，于是想去掉所有再增加我们自己的
`-e CUSTOM_MODELS:-all,+SparkDesk `
但是实际并没有什么改变，请各位大大帮忙看看。谢谢
![图片](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/31498851/c95d3f8b-3640-4175-bf48-4ca5c2cfb34d)

**Comment by ersaijun** [comment]
-e CUSTOM_MODELS=-all,+SparkDesk 
你写错了
